ServerCore/Startup.cs

```diff
@@ -117,7 +117,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 app.UseHsts();
                 PuzzleServerContext.UpdateDatabase(app);
             }
-            else if (env.IsProduction() && (Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzlehunt" || Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzleday"))
+            else if (env.IsProduction() && (Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzlehunt" || Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzleday" || Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzlesafari"))
```

Comment:
vestigial

Reply:
Reverted.

---
In reply to: [295581687](https://github.com/PuzzleServer/mainpuzzleserver/pull/491#discussion_r295581687) [](ancestors = 295581687)